log/logger.log

```diff
@@ -0,0 +1,41 @@
+16:41:36,142 INFO  MyThread:17 - 1
+ 16:41:36,143 INFO  MyRunnable:17 - 2
+ 16:41:36,147 INFO  MyThread:17 - 3
+ 16:41:36,148 INFO  MyRunnable:17 - 4
+ 16:41:36,148 INFO  MyThread:17 - 5
+ 16:41:36,148 INFO  MyThread:17 - 7
+ 16:41:36,148 INFO  MyRunnable:17 - 6
+ 16:41:36,148 INFO  MyThread:17 - 8
+ 16:41:36,152 INFO  MyRunnable:17 - 9
+ 16:41:36,152 INFO  MyThread:17 - 10
+ 16:41:36,152 INFO  MyRunnable:17 - 11
+ 16:41:36,152 INFO  MyThread:17 - 12
+ 16:41:36,153 INFO  MyRunnable:17 - 13
+ 16:41:36,153 INFO  MyThread:17 - 14
+ 16:41:36,154 INFO  MyRunnable:17 - 15
+ 16:41:36,154 INFO  MyThread:17 - 16
+ 16:41:36,154 INFO  MyRunnable:17 - 17
+ 16:41:36,156 INFO  MyThread:17 - 18
+ 16:41:36,156 INFO  MyRunnable:17 - 19
+ 16:41:36,157 INFO  MyThread:17 - 20
+ 16:41:36,157 INFO  MyRunnable:17 - 21
+ 16:41:36,157 INFO  MyRunnable:17 - 23
+ 16:41:36,157 INFO  MyRunnable:17 - 24
+ 16:41:36,157 INFO  MyRunnable:17 - 25
+ 16:41:36,158 INFO  MyRunnable:17 - 26
+ 16:41:36,158 INFO  MyRunnable:17 - 27
+ 16:41:36,157 INFO  MyThread:17 - 22
+ 16:41:36,158 INFO  MyRunnable:17 - 28
+ 16:41:36,158 INFO  MyThread:17 - 29
+ 16:41:36,158 INFO  MyRunnable:17 - 30
+ 16:41:36,158 INFO  MyThread:17 - 31
+ 16:41:36,159 INFO  MyThread:17 - 33
+ 16:41:36,159 INFO  MyRunnable:17 - 32
+ 16:41:36,159 INFO  MyThread:17 - 34
+ 16:41:36,159 INFO  MyRunnable:17 - 35
+ 16:41:36,159 INFO  MyThread:17 - 36
+ 16:41:36,160 INFO  MyRunnable:17 - 37
+ 16:41:36,160 INFO  MyThread:17 - 38
+ 16:41:36,160 INFO  MyThread:17 - 39
+ 16:41:36,160 INFO  MyThread:17 - 40
+ 
```

Comment:
missing empty line at the end of the file

Reply:
fixed